webapp/src/index.js

```diff
@@ -15,13 +15,25 @@ import Client from './client';
 class Plugin {
     // eslint-disable-next-line no-unused-vars
     initialize(registry, store) {
-        registry.registerChannelHeaderButtonAction(
-            <Icon/>,
-            (channel) => {
-                startMeeting(channel.id)(store.dispatch, store.getState);
-            },
-            'Start Webex Meeting'
-        );
+        const helpText = 'Start Webex Meeting';
+        const action = (channel) => {
+            startMeeting(channel.id)(store.dispatch, store.getState);
+        };
+
+        // Channel header icon
+        registry.registerChannelHeaderButtonAction(<Icon/>, action, helpText);
+
+        // App Bar icon
+        if (registry.registerAppBarComponent) {
+            let siteUrl = '';
+            const config = getConfig(store.getState());
+            if (config) {
+                siteUrl = config.SiteURL;
```

Comment:
@mickmister, using `?.` gives an error, since this is a javascript file. Could you take another quick look at the solution? https://github.com/mattermost/mattermost-plugin-webex/pull/68/commits/d05d62accd017dd9e486f38ac8f18150c2a423b1 Thanks!

```
SyntaxError: /home/alejandro/projects/mattermost/mattermost-plugin-webex/webapp/src/index.js: Support for the experimental syntax 'optionalChaining' isn't currently enabled (29:35):
```